wodin_deploy/wodin_constellation.py

```diff
@@ -0,0 +1,117 @@
+from constellation import Constellation, ConstellationContainer, ConstellationMount, docker_util
+
+
+class WodinConstellation:
+    def __init__(self, cfg):
+        self.cfg = cfg
+        redis = redis_container(cfg)
+        odin_api = odin_api_container(cfg)
+        sites = sites_containers(cfg)
+        proxy = wodin_proxy_container(cfg)
+
+        containers = [redis, odin_api, *sites, proxy]
+
+        self.obj = Constellation(
+            "wodin",
+            cfg.container_prefix,
+            containers,
+            cfg.network,
+            {"redis-data": "redis-data", "wodin-config": "wodin-config"},
+            data=cfg,
+        )
+
+    def start(self, **kwargs):
+        self.obj.start(**kwargs)
+
+    def stop(self, **kwargs):
+        self.obj.stop(**kwargs)
+
+    def status(self):
+        self.obj.status()
+
+
+def redis_container(cfg):
+    redis = cfg.redis
+    redis_mounts = [ConstellationMount("redis-data", "/data")]
+    return ConstellationContainer(redis["name"], redis["ref"], mounts=redis_mounts, network=cfg.network)
+
+
+def odin_api_container(cfg):
+    odin_api = cfg.odin_api
+    return ConstellationContainer("api", odin_api["ref"], network=cfg.network)
```

Comment:
this the only name hardcoded because i wanted to call it "api" :laughing:. If we mess with the name in the config, it will change the docker image reference, wasn't sure if this was worth abstracting

Reply:
Sorry can you explain that a bit more? I would expect changing the name in the config to update the docker image reference name. Is there a reason you want that to be static though?

You re-use this hardcoded "api" name later, might be worth moving that into the building of the config so it is just in one place?

Reply:
I see thanks! Makes sense to have the easier to write name. I see you've updated that, that looks good :)